research-guide-keywords.php

```diff
@@ -152,9 +152,9 @@
 if ($reskeyword == $termslug){
 
     // echo '<li><b>'. $termname. '</b></li>';
-    echo '<li><a href="?show=keywords&keyword-letter='.$reskeyletter.'&keyword='.$termslug.'#step-three" autofocus><strong>'. $termname. '</strong></a></li>';
+    echo '<li><a aria-selected="true" href="?show=keywords&keyword-letter='.$reskeyletter.'&keyword='.$termslug.'#step-three" autofocus><strong>'. $termname. '</strong></a></li>';
```

Comment:
I know this is exactly what DAC suggested but I think this is wrong.

`aria-selected` is for certain role types such as tabs: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-selected

I think what we might actually need is `aria-current` which is more for this scenario: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-current

I wonder if we can talk to DAC about this?

Reply:
Good call @ahosgood , `aria-current `looks to be more relevant here. I'm happy to change it and we can always check with DAC when we have the opportunity.